apps/server/src/modules/files/files.module.ts

```diff
@@ -1,12 +1,22 @@
 import { Module } from '@nestjs/common';
 import { FileStorageAdapter } from '@shared/infra/filestorage';
-import { FilesRepo, StorageProviderRepo } from '@shared/repo';
+import { FileRecordRepo, FilesRepo, StorageProviderRepo, TaskRepo } from '@shared/repo';
 import { LoggerModule } from '@src/core/logger';
 import { DeleteFilesConsole } from './job/delete-files.console';
+import { SyncFilerecordsConsole } from './job/sync-filerecords.console';
 import { DeleteFilesUc } from './uc';
 
 @Module({
 	imports: [LoggerModule],
-	providers: [DeleteFilesConsole, DeleteFilesUc, FilesRepo, StorageProviderRepo, FileStorageAdapter],
+	providers: [
+		DeleteFilesConsole,
```

Comment:
I understand why the job is added to the model, but in the end when we finished, we must remove or delete it from this place.